grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -320,14 +320,14 @@ class InventoryItemController {
         requisitionItems = requisitionService.getPendingRequisitionItems(location, product)
         requisitionItems.groupBy { it.requisition }.collect { k, v ->
             itemsMap.put(k, [
+                    requisitionItems: requisitionService.getListOfDetailsForPendingOutbound(v),
                     quantityRequested: v.quantity.sum(),
                     quantityRequired: v.sum() { RequisitionItem requisitionItem -> requisitionItem.calculateQuantityRequired() },
                     quantityPicked: v.sum() { RequisitionItem requisitionItem -> requisitionItem.calculateQuantityPicked() },
```

Comment:
if it's not needed anymore you can remove it

Reply:
or change the quantityPicked to quantityPickedByLot

Reply:
It is sum up qty picked for requisition and is used in another place for summing this values at a bottom of a table

Reply:
ok